pkg/meta/meta.go

```diff
@@ -63,6 +65,18 @@ func ReferenceTo(o metav1.Object, of schema.GroupVersionKind) *corev1.ObjectRefe
 	}
 }
 
+// TypedReferenceTo returns a typed object reference to the supplied object,
+// presumed to be of the supplied group, version, and kind.
+func TypedReferenceTo(o metav1.Object, of schema.GroupVersionKind) *v1alpha1.TypedReference {
```

Comment:
Should we mark the existing `ReferenceTo` implementation deprecated, in service of https://github.com/crossplane/crossplane-runtime/issues/49?

Reply:
Good idea! Will update :+1: